src/assets/wise5/components/ComponentInfo.ts

```diff
@@ -0,0 +1,12 @@
+export abstract class ComponentInfo {
+  abstract description: string;
+  abstract previewContent: any;
```

Comment:
Can these be declared as protected fields? Also in all the child classes of ComponentInfo?
```suggestion
  protected abstract description: string;
  protected abstract previewContent: any;
```